WooCommerce/Classes/ViewRelated/Dashboard/Blaze/BlazeCampaignDashboardViewModel.swift

```diff
@@ -18,7 +19,7 @@ final class BlazeCampaignDashboardViewModel: ObservableObject {
 
     @Published private(set) var state: State
 
-    @Published private(set) var shouldShowInDashboard: Bool = false
+    @Published private(set) var shouldShowInDashboard: Bool = true
```

Comment:
I'm curious why we need this change?

Reply:
Removed in https://github.com/woocommerce/woocommerce-ios/pull/10962/commits/8d9073362885751f089222e56458f170997af708